OneCommunityGlobal/HighestGoodNetworkApp
# Lucile information points on everything of interest

# Description
3. (PRIORITY HIGH) Jae: Make everything of interest have an “i” by it explaining how it works (WIP - Lucile) 
Need someone to through the whole app and create a spreadsheet of every area that doesn’t already have an “i” by it that should. Then help get those all added! 
Organize the spreadsheet (or Google Doc) with the same sections you see on this doc
Include in the doc pictures of where you plan to put each “i” so I can approve them
To keep the app simple for the most common user class of Volunteers, also include/decide which “i”s should be visible to everyone but Volunteers
Make those “i”s blue (#00CCFF) so it’s clear it is a special thing visible only to certain classes
Make all “i”s editable by Owner class so changes in use, meaning, etc. can easily be updated by the Owner


## Main changes explained:
Here is the link to the Google Doc for info points management: https://docs.google.com/document/d/1f2s1Y7oivAqgwDLBjALtOloIcR0hQ3iAEw5l2v5eZBo/edit?usp=sharing

This document contains: 

Existing info points: Historical info points present in production that have been made editable by owners in this PR

Non-Editable/Mouse-hover info points: Info points that appear with a mouse hover and therefore are not modified in this PR.

New info points added: New editable info points added in this PR

New info point ideas: Please feel free to put your info point ideas in this Google Doc


## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as an owner user
4. go to the different pages to see if the info points are editable when clicking on it
5. log as admin user
6. go to the different pages to see if the info points are displayed but not editable
7. log as regular user
8. go to the different pages to see if the info points are displayed but not editable
6. Please, add info point ideas in [this Google Doc
](https://docs.google.com/document/d/1f2s1Y7oivAqgwDLBjALtOloIcR0hQ3iAEw5l2v5eZBo/edit?usp=sharing)if you feel that some parts of the app have to have an info point


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111132148/5ad5d569-188c-411d-bb01-fdf63686ec2f
